andela/mazus-ah-backend
# #167165055 Send verification email

### What does this PR do?
Creates a functionality that sends verification link via email to a user upon successful registration

#### Description of Task to be completed?
- Given that a user successfully signs up
- The user is sent a verification link via email
- Upon clicking the link, the user is verified 

#### How should this be manually tested?
1. Fetch this branch on the terminal using 
`git fetch origin ft-send-verification-email-167165055`
2. Install dependencies using `npm install`
3. Start the server with `npm run server`
4. On Postman, sign up by making a POST request to /api/v1/auth/signup.
5. Check the inbox of the email you signed up with and click the link sent.
6. Upon redirection to the browser, copy the link in the URL bar.
7. Paste this link in Postman and make a PATCH request.


#### What are the relevant pivotal tracker stories?
[#167165055](https://www.pivotaltracker.com/story/show/167165055)

#### Screenshots (if appropriate)
<img width="820" alt="Screen Shot 2019-07-27 at 9 36 07 AM" src="https://user-images.githubusercontent.com/41482184/61992299-3dacff80-b054-11e9-84dc-b259a5fed1ee.png">
<img width="1157" alt="Screen Shot 2019-07-27 at 9 50 20 AM" src="https://user-images.githubusercontent.com/41482184/61992313-659c6300-b054-11e9-8622-872ecd774844.png">
